server/lib/utils.js

```diff
@@ -492,7 +492,8 @@ export const cleanTags = tags => {
   const cleanTagsList = tags
     .filter(t => Boolean(t)) // Remove null values
     .map(t => t.trim()) // Trim tags
-    .filter(t => t.length > 0); // Remove empty tags
+    .filter(t => t.length > 0) // Remove empty tags
+    .map(t => t.replace(/^#/g, '')); // Remove # prefixes
```

Comment:
If we do this after the `filter`, then we can end up with empty tags. Suggested:

```suggestion
    .map(t => t.replace(/^#+/g, '')); // Remove # prefixes
    .map(t => t.trim()) // Trim again for empty tags with a # prefix
    .filter(t => t.length > 0) // Remove empty tags
```

Would be great to add some tests in `test/server/lib/utils.test.js` while we're at it.

Reply:
Done, and I've added a test as well. 👍🏽